### DISCUSSION TITLE
I have to restart after every suggestion

### DISCUSSION THREAD

**Discussion by pheroux** [community/community#19221] [Copilot Conversations]
I'm using Visual Studio and I have to restart after every suggestion.
It only gives me a single suggestion before it stops working. Then, I have to restart Visual Studio.

What gives?

**Comment by melvincwng** [comment]
@pheroux, can you try 'Cltrl' + 'Enter' in VSCode?

This is the command for Github Copilot that gives multi line suggestions (if available).

Hope this solution works for you, at least its better than restarting VSCode after every suggestion.

**Reply by pheroux** [reply]
I guess I needed to enter my payment information for copilot to be truly enabled.
But now I have a different problem.

When copilot starts I get this error:
`[Error] An error occurred for 'LSP': StreamJsonRpc.RemoteInvocationException: Failed to start host 'node.x86$XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'. Host exited with exit code '1'. (ErrorKind: Error)`

I tried reinstalling the extension but no luck.

**Reply by melvincwng** [reply]
@pheroux, seems like there's quite a number of threads/discussions regarding issues on Github Copilot recently.

If all fails, perhaps try raising a ticket to [GitHub Support](https://support.github.com/contact/account).

All the best.
